config/install/user.role.anonymous.yml

```diff
@@ -10,5 +10,6 @@ permissions:
   - 'access content'
   - 'access site-wide contact form'
   - 'search content'
+  - 'use exclude node title'
```

Comment:
I think we may also need to add this `'use exclude node title'` permision to the authenticated user role.